torchprime/torch_xla_models/configs/default.yaml

```diff
@@ -14,7 +14,8 @@ logging_steps: 10
 torch_dtype: bfloat16
 
 # This might be overwritten when using tp run to launch the run using XPK
```

Comment:
I thinks this comment is misplaced. It should be on `profile_dir`. (Technically not part of your PR but since you're here)